### PR TITLE
React 18 fix

### PR DESCRIPTION
**Pull request by justinwatkinsact**
Updated deps, tests and added proper base class for react 18 children property on ThemeProvider

### CODE REVIEW

File: src/components/IdleTimer/index.tsx [hunk 1]
```diff
@@ -47,9 +47,9 @@ export const IdleTimer: React.FC<IdleTimerProps> = ({
         onActive(event);
       }
     },
-    onIdle: (event): void => {
+    onIdle: (): void => {
```
Comment (joncursi) [inline src/components/IdleTimer/index.tsx:50]: Is this a breaking change?

**Review by joncursi — APPROVED** [review]
All looks good to me, just not sure if the idle timer changes are breaking or not. 

Could you also add in a changelog entry + version bump, or are those 100% automated now as part of the release process?

File: src/components/IdleTimer/index.tsx (hunk 1 above)
Comment (justinwatkinsact) [reply src/components/IdleTimer/index.tsx:50]: @joncursi , it shouldn't be.  That library doesn't return anything here.  With the typescript / lib update, it caught that as a type error.

**Comment by justinwatkinsact** [comment]
@joncursi, release notes should all be automated.  Once I publish, it should add that.  I will make sure.

**Pull request merged**